tests/event_lists/test_3.py

```
@@ -5,12 +5,49 @@
-    "south korean presidential election 2017": "الانتخابات الرئاسية الكورية الجنوبية 2017",
-    "republic of ireland women's national football team": "منتخب جمهورية أيرلندا لكرة القدم للسيدات",
 }
 
 data_1 = {
+    "People from the Spanish Netherlands executed for witchcraft": "أشخاص من هولندا الإسبانية أعدموا بتهمة السحر",
+    "French people executed by guillotine during the French Revolution": "فرنسيون أعدموا بالمقصلة خلال الثورة الفرنسية",
+    "People executed by Kingdom of England by burning": "أشخاص أعدموا من قبل مملكة إنجلترا بالحرق",
+    "People executed by the House of Lancaster": "أشخاص أعدموا من قبل الهاوس في لانكستر",
+    "People executed by the House of York": "أشخاص أعدموا من قبل الهاوس في يورك",
+    "People executed by the International Military Tribunal in Nuremberg": "أشخاص أعدموا من قبل المحكمة العسكرية الدولية في نورنبيرغ",
+    "People executed by the Jin dynasty (266–420) by decapitation": "أشخاص أعدموا من قبل أسرة جين (266–420) بقطع الرأس",
```

Ruff (0.14.13)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)